src/templates/header.html

```diff
@@ -22,6 +33,19 @@ <h1 class="site">
   <button class="mkt-nav--toggle hamburger" data-toggle-nav="nav"
           title="{{ _('Toggle Menu') }}"></button>
 
+  <h1 class="site">
+    <a href="{{ url('homepage') }}" class="wordmark"><span>Firefox Marketplace</span></a>
+    <a href="{{ url('homepage') }}" class="site-name">Marketplace</a>
+  </h1>
+
+  <mkt-header-nav>
+    {{ mkt_header_nav_item(url('homepage'), _('Home'), 'homepage') }}
+    {{ mkt_header_nav_item(url('popular'), _('Popular'), 'popular') }}
+    {{ mkt_header_nav_item(url('new'), _('New'), 'new') }}
+    {{ mkt_header_nav_item(url('recommended'), _('Recommended'), 'recommended') }}
+    {{ mkt_header_nav_item('#', _('Categories'), 'categories', 'categories') }}
```

Comment:
This may as well link to `/categories`, I'd think. Although that page should probably die (and will likely stop working when we're done if we don't fix it).